sys-fabric/libibumad/libibumad-1.3.10.2.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="This is a sample skeleton ebuild file"
+HOMEPAGE="https://www.openfabrics.org"
+SRC_URI="https://www.openfabrics.org/downloads/management/${P}.tar.gz"
+
+SLOT="0"
+
+KEYWORDS="amd64"
```

Comment:
In new versions it must be unstable, meaning `~amd64`.
Is only `amd64` supported now? I think we can put here the unstable version of the old ebuild, meaning:
`KEYWORDS="~amd64 ~x86 ~amd64-linux"`